types/vote_set.go

```diff
@@ -133,8 +134,13 @@ func (voteSet *VoteSet) AddVote(vote *Vote) (added bool, err error) {
 	return voteSet.addVote(vote)
 }
 
+var errNilVote = errors.New("nil/non-dereferenceable votes are not allowed")
```

Comment:
maybe we can put this in vote.go with the other errors

Reply:
Roger that, thanks.